core-server/src/main/java/org/orbisgis/orbisserver/IndexController.java

```
@@ -61,46 +66,62 @@
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.DataInputStream;
+import java.io.File;
 import java.io.InputStream;
 import java.util.List;
+import java.util.ArrayList;
 
 /**
  * Very light instance of DefaultController containing a WpsServer.
  *
  * @author Sylvain PALOMINOS
+ * @author Guillaume MANDE
  */
 @Controller
 public class IndexController extends DefaultController {
 
-    /**
-     * Gets the instance of the WpsServer.
-     */
+    /** Gets the instance of the WpsServer. */
     private WpsServer wpsServer = WpsServerManager.getWpsServer();
 
+    /** List all of the OrbisWPS processes into a String. */
+    private String test = "";
+
+    /**  Used to display the corresponding xml file to GetCapabilities method. */
+    public WPSCapabilitiesType wpsCapabilitiesType = null;
+
+    /** Used by methods like DescribeProcessRequest to get all the ProcessSummaryType */
+    public List<ProcessSummaryType> list = null;
+
+    /** List of all identifier of all the processes */
+    private List<CodeType> codeTypeList = null;
+
     /**
      * Injects a template named 'index'.
      */
     @View("index")
     Template index;
 
+
+
     /**
-     * The action method returning the index page. It handles
-     * HTTP GET request on the "/" URL.
+     * The action method returning the html index page containing a list of all the OrbisWPS processes
+     * readable by a human. It handles HTTP GET request on the "/index" URL.
      *
-     * @return the index page
      */
-    @Route(method = HttpMethod.GET, uri = "/")
+    @Route(method = HttpMethod.GET, uri = "/index")
     public Result index() {
         //Simple example of getting information from the WpsServer
-        try { simpleWpsRequest(); } catch (JAXBException ignored) {}
-        return ok(render(index, "index", "Welcome"));
+        try { simpleWpsRequest();  } catch (JAXBException ignored) {}
```

Avoid to ignore an exception. At least log a message.